### PR TITLE
[WebUI] Support returning a 2xx status code for non-existing document rows

### PR DESCRIPTION
**Pull request by teosarca**
https://github.com/metasfresh/metasfresh/issues/6838

### CODE REVIEW

**Comment by teosarca** [comment]
image

**Comment by siemiatj** [comment]
Adding new unit tests for the frontend part but can merge them separately.

File: backend/metasfresh-webui-api/src/test/java/de/metas/ui/web/window/datatypes/json/JSONDocumentListTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	private ObjectMapper jsonObjectMapper;
+
+	@BeforeEach
+	public void init()
+	{
+		jsonObjectMapper = JsonObjectMapperHolder.newJsonObjectMapper();
+	}
+
+	@Test
+	public void testSerialize()
```
Comment (TheBestPessimist) [inline backend/metasfresh-webui-api/src/test/java/de/metas/ui/web/window/datatypes/json/JSONDocumentListTest.java:47]: shoulnd't we have a test with some data in the document list as well?

**Comment by siemiatj** [comment]
Investigating some failed e2e tests: https://dashboard.cypress.io/projects/5yp4q1/runs/796/specs

**Comment by siemiatj** [comment]
Fixed failing e2e tests

**Pull request merged**